spec/system/backend/solidus_stripe/orders/payments_spec.rb

```diff
@@ -0,0 +1,111 @@
+# frozen_string_literal: true
+
+require 'solidus_stripe_spec_helper'
+
+RSpec.describe 'SolidusStripe Orders Payments', :js do
+  include SolidusStripe::BackendTestHelper
+  stub_authorization!
+
+  context 'with successful payment operations' do
+    it 'navigates to the payments page' do
+      payment = creates_authorized_payment
+      visits_payments_page
+
+      payment_displayed(payment)
+    end
+
+    it 'displays log entries for a payment' do
+      payment = creates_authorized_payment
+      message = "authorized payment intent"
+      creates_payment_log_entry(payment, message)
```

Comment:
Thoughts on using factories for `creates_authorized_payment` and `creates_payment_log_entry`?
e.g. `create(:stripe_authorized_payment)`

Reply:
✔️  I tried to keep the factories generic for possible other future use, let me know what you think